langs/python.go

```diff
@@ -10,6 +10,24 @@ type PythonLangHelper struct {
 	Version string
 }
 
+func (h *PythonLangHelper) Handles(lang string) bool {
+	return defaultHandles(h, lang)
+}
+func (h *PythonLangHelper) Runtime() string {
+	return h.LangStrings()[0]
+}
+
+// TODO: I feel like this whole versioning thing here could be done better. eg: `runtime: python:2` where we have a single lang, but support versions in tags (like docker tags).
+func (lh *PythonLangHelper) LangStrings() []string {
```

Comment:
This feels a bit messy (as h.Version is already specified on construction) - right now  I prefer if we set the runtime name (for java and python)  (and maybe aliases) on struct init ? thoughts?

Reply:
Agree, feels very messy (see my comment). 

I think we should have tags for versions rather than explicit runtimes with the versions built in. eg: `runtime: java:9` . runtime is java, version is 9. This would apply to all languages for versioning. 

I didn't attempt fixing that issue in this PR though. It uses a similar (bad) scheme as before: https://github.com/fnproject/cli/pull/118/files#diff-a3639713bb27239dd39d24f7d2fc9678L25, but still fixes the discrepancy issues. 

Made a follow up #119 to do the version thing properly.

Reply:
ack, thanks